xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/VideoLayerBridgeDRMPRIME.cpp

```diff
@@ -174,6 +200,38 @@ void CVideoLayerBridgeDRMPRIME::Configure(CVideoBufferDRMPRIME* buffer)
     m_DRM->AddProperty(plane, "COLOR_RANGE", value);
 
   auto connector = m_DRM->GetConnector();
+
+  int bpc = 8;
+  bool ycc = false;
+  std::string output_format = "Default";
+
+  if (picture.colorBits > 8)
+  {
+    bpc = 12;
+    ycc = true;
+    output_format = "YCrCb422";
```

Comment:
How can we know to use this format specifically? Does the driver give any feedback about what to use or do you just have to know?

What about 444 or 420?

Reply:
The version of the kernel patches I have been working on recently will choose RGB by default, and if it doesn't succeed (with a clock too high), try YUV422. So the userspace doesn't have to know anything.